docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -112,6 +112,7 @@ $ oc set data secret/osp-secret "HeatPassword=$HEAT_PASSWORD"
 $ oc set data secret/osp-secret "IronicPassword=$IRONIC_PASSWORD"
 $ oc set data secret/osp-secret "IronicInspectorPassword=$IRONIC_PASSWORD"
 $ oc set data secret/osp-secret "ManilaPassword=$MANILA_PASSWORD"
+$ oc set data secret/osp-secret "MetadataSecret=$METADATA_SECRET"
 $ oc set data secret/osp-secret "NeutronPassword=$NEUTRON_PASSWORD"
 $ oc set data secret/osp-secret "NovaPassword=$NOVA_PASSWORD"
```

Comment:
Hackfest comment for reference: https://docs.google.com/document/d/17GzJf8uTfpe7TJrEpf9NomZ98LcnNvUs2V3j9jgDw2w/edit?disco=AAABRuPpC2k